### DISCUSSION TITLE
[ FF7 ] Game crashes shortly after executing (OpenGL)

### DISCUSSION THREAD

**Discussion by GrumpyGourmand** [julianxhokaxhiu/FFNx#231] [Q&A] [answered]
**Describe the bug**
Upon double-clicking the ff7.exe executable, it crashes after a few seconds and nothing ever appears on the screen other than black.

**To Reproduce**

1. Set rendering backend to OpenGL (or anything other than Direct3D9) in FFNx.toml
2. Execute ff7.exe
3. Witness crash

**Expected behavior**
The game runs and you see the Eidos video playing.

**GPU (please complete the following information):**

- Intel HD Graphics (Core i5 1st gen)
- Driver Version 8.15.10.2900

**Additional context**
No 7th Heaven, just FFNx 1.8.1.0, vanilla game with no mods.

I have tried every rendering backend, and all of them crash in the same manner, except Direct3D9, which looks and runs poorly.  I know this iGPU is old and slow, but OpenGL works fine with Aali's driver.
[FFNx.log](https://github.com/julianxhokaxhiu/FFNx/files/6345786/FFNx.log)

**Comment by julianxhokaxhiu** [comment, marked as answer]
Hi, I'm sorry to inform you that your GPU might not be capable of running the game using FFNx as it requires a modern GPU. Other than suggesting you to update the drivers, I can't do much sorry. Try with another GPU if you do have one. Cheers!
